backends/rapidpro/backend_test.go

```diff
@@ -777,39 +777,33 @@ func (ts *BackendTestSuite) TestDupes() {
 
 	msgJSON, err := json.Marshal([]interface{}{dbMsg})
 	ts.NoError(err)
-
 	err = queue.PushOntoQueue(r, msgQueueName, "dbc126ed-66bc-4e28-b67b-81dc3327c95d", 10, string(msgJSON), queue.HighPriority)
 	ts.NoError(err)
-
 	_, err = ts.b.PopNextOutgoingMsg(ctx)
 	ts.NoError(err)
 
-	msg = ts.b.NewIncomingMsg(knChannel, urn, "test", clog).(*DBMsg)
-	err = ts.b.WriteMsg(ctx, msg, clog)
-	ts.NoError(err)
-
-	ts.NotEqual(uuid2, msg.UUID())
-}
-
-func (ts *BackendTestSuite) TestExternalIDDupes() {
-	r := ts.b.redisPool.Get()
-	defer r.Close()
+	msg4 := createAndWriteMsg(knChannel, urn, "test", "")
```

Comment:
Can we also have test of a message with the same text but now with an attachment or 2 messages with the same text but different attachments? without external IDs?

That is the cases I think we miss only from these changes